api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1414,22 +1414,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.isRetired());
-		assertEquals("Some Retire Reason", orderFrequency.getRetireReason());
```

Comment:
This is problematic because you are changing the expected behaviour. The assertion ensured that the retire reason should of an expected text, which is different from just ensuring that it is not null.